sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -140,9 +140,27 @@ public void validate(PipelineOptions options) {}
   public void populateDisplayData(DisplayData.Builder builder) {
     super.populateDisplayData(builder);
 
+    // Append wildcard to browseUrl input since this is a filename prefix
+    String browseUrl = null;
+    String browseFilePattern = baseOutputFilename + "*";
```

Comment:
This logic has some issues:
* If the prefix is just a bucket name, GcsPath will throw
* If it is a local file pattern, adding a glob pattern doesn't make sense.